nan.h

```diff
@@ -173,7 +173,7 @@ NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
 }
 
 /* io.js 1.0  */
-#if NODE_MODULE_VERSION >= IOJS_1_0_MODULE_VERSION \
+#if NODE_MODULE_VERSION >= NODE_0_12_MODULE_VERSION \
  || NODE_VERSION_AT_LEAST(0, 11, 15)
```

Comment:
/cc @rogerwang - how do we (nan, nw.js) resolve the above now and in the future?